content/home.md

```diff
@@ -27,23 +27,10 @@ Online ∙ Print ∙ E‑book
 
 </div>
 
-Quire is a modern, multiformat publishing tool designed for longevity, discoverability, and scholarship. Using a single set of plain text files, Quire creates books as authoritative and enduring as print and as vibrant and feature-rich as the web—all without paying a fee or maintaining a complicated server.
+Quire is an open-source multiformat publishing tool designed for longevity, discoverability, and scholarship. Using a single set of plain text files, Quire creates books as authoritative and enduring as print and as vibrant and feature-rich as the web—all without paying a fee or maintaining a complicated server.
 
 Created by [Getty](https://www.getty.edu), free to use.
 
-<div class="action-button with-sub-action-button">
-
-[Sign-Up for Quire](https://docs.google.com/forms/d/e/1FAIpQLScKOJEq9ivhwizmdazjuhxBII-s-5SUsnerWmyF8VteeeRBhA/viewform)
-
-</div>
-
-<div class="sub-action-button">
-
-[View on GitHub ![logo](/img/icons/github.svg) ](https://github.com/thegetty/quire/)
-
-</div>
```

Comment:
What do you think about leaving these in but having the button be "Get Started"? It's duplicative of the item in the top menu, but I think it's a natural place for people to look and to find a call to action. I also think there's value in still having the GitHub link up there.